llvm/include/llvm/CodeGen/MachinePipeliner.h

```diff
@@ -294,6 +294,8 @@ class SwingSchedulerDAG : public ScheduleDAGInstrs {
 
   static bool classof(const ScheduleDAGInstrs *DAG) { return true; }
 
+  bool mayOverlapInLaterIter(const MachineInstr *MIA, const MachineInstr *MIB);
```

Comment:
Can you please unify the names of the arguments?

```suggestion
  bool mayOverlapInLaterIter(const MachineInstr *BaseMI,
                             const MachineInstr *OtherMI);
```

Reply:
Fixed